ecs-cli/modules/cli/compose/project/project_parseV1V2_test.go

```diff
@@ -0,0 +1,377 @@
+package project
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"testing"
+
+	"github.com/aws/amazon-ecs-cli/ecs-cli/modules/cli/compose/containerconfig"
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/ecs"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestParseV1V2_Version1_HappyPath(t *testing.T) {
+	// expected ContainerConfig values
+	redisImage := "redis"
+	webImage := "web"
+	cpuShares := int64(73)
+	command := []string{"bundle exec thin -p 3000"}
+	dnsServers := []string{"1.2.3.4"}
+	dnsSearchDomains := []string{"search.example.com"}
+	entryPoint := []string{"/code/entrypoint.sh"}
+	env := []*ecs.KeyValuePair{
+		{
+			Name:  aws.String("RACK_ENV"),
+			Value: aws.String("development"),
+		},
+	}
+	extraHosts := []*ecs.HostEntry{
+		{
+			Hostname:  aws.String("test.local"),
+			IpAddress: aws.String("127.10.10.10"),
+		},
+	}
+	hostname := "foobarbaz"
+	labels := map[string]*string{
+		"label1":         aws.String(""),
+		"com.foo.label2": aws.String("value"),
+	}
+	links := []string{"redis:redis"}
+	logDriver := aws.String("json-file")
+	logOpts := map[string]*string{
+		"max-file": aws.String("50"),
+		"max-size": aws.String("50k"),
+	}
+	logging := &ecs.LogConfiguration{
```

Comment:
you're defining `logging` here but not using it in the test yaml string below (which makes sense, since this field wasn't defined for V1)
https://docs.docker.com/compose/compose-file/compose-file-v1/#log_driver

Reply:
I don't care about what it looks like coming in or how it even looks like on the libcompose ServiceConfig -- I only care that it's being set correctly on ContainerConfig, which is why I wrapped the log options and driver in the `LogConfiguration` struct. The previous test were misleading/not the most useful bc it was just testing libcompose parsing behavior.

Reply:
ah, I was definitely confusing some of the previous test stuff with this - of course the resulting containerConfig would have an ecs.LogConfiguration given drivers/options were in the YAML.